app/views/templates/_form.html.erb

```diff
@@ -199,4 +210,5 @@
 
     <%= submit_or_cancel f, false, :cancel_path => template_path_for(@template.class) %>
   </div>
+  <%= f.hidden_field :cloned_from_id %>
```

Comment:
What is this good for? Can I change this internal value if I modify this in browser console?

Reply:
It's probably a relic from ancient times when I needed it for the implementation, and then I went with a different approach.

I'll remove it.